share/availability.go

```diff
@@ -23,3 +33,13 @@ type Availability interface {
 	// TODO(@Wondertan): Merge with SharesAvailable method, eventually
 	ProbabilityOfAvailability() float64
 }
+
+// translate transforms square coordinates into IPLD NMT tree path to a leaf node.
+// It also adds randomization to evenly spread fetching from Rows and Columns.
+func Translate(dah *Root, row, col int) (cid.Cid, int) {
```

Comment:
I've placed `Root` here because it is the only place it is used in the `share` pkg. If there is a need to get info about `Root`, it is always in context of `share.Avalability`, that is residing nearby. I feel like having it grouped is increasing variable locality and reducing cognitive load.
 On the other hand `Share` is more widely used and would be nice to have it more atomic.

Reply:
Move to ipld now

Reply:
moved to `ipld` pkg